src/util.ts

```diff
@@ -209,10 +209,13 @@ export async function emsa_pss_encode(
 
 // RSAVP1
 // https://www.rfc-editor.org/rfc/rfc3447.html#section-5.2.2
-export function rsavp1(pkS: { n: sjcl.bn; e: sjcl.bn }, s: sjcl.bn): sjcl.bn {
+export function rsavp1(
+    pkS: { n: sjcl.BigNumber; e: sjcl.BigNumber },
+    s: sjcl.BigNumber,
+): sjcl.BigNumber {
     //  1. If the signature representative s is not between 0 and n - 1,
     //    output "signature representative out of range" and stop.
-    if (!s.greaterEquals(new sjcl.bn(0)) || s.greaterEquals(pkS.n) == 1) {
+    if (!s.greaterEquals(new sjcl.bn(0)) || s.greaterEquals(pkS.n)) {
```

Comment:
Should this be `BigNumber` here?
```suggestion
    if (!s.greaterEquals(new sjcl.BigNumber(0)) || s.greaterEquals(pkS.n)) {
```

Reply:
when passing a number, the function internally creates a new object, then it becomes equivalent.